aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -190,9 +190,12 @@ def __init__(
         allow_credentials: bool
             A boolean value that sets the value of `Access-Control-Allow-Credentials`
         """
-        self._allowed_origins = [allow_origin]
+
+        self.allowed_origins = [allow_origin]
```

Comment:
I'm not very comfortable with that. I know that we are accessing this property in another class and because it should be public, but we made it private to reduce the API surface of this class and the customer does not inadvertently try to modify this property.

But I agree with whatever you decide.